lakekeeper/project.go

```
@@ -144,6 +153,26 @@ func (s *ProjectService) CreateProject(opts *CreateProjectOptions, options ...Re
 	return project, resp, nil
 }
 
+// RenameProject renames a project.
+//
+// Lakekeeper API docs:
+// https://docs.lakekeeper.io/docs/nightly/api/management/#tag/project/operation/rename_project
+func (s *ProjectService) RenameProject(id string, opts *RenameProjectOptions, options ...RequestOptionFunc) (*http.Response, error) {
+	options = append(options, WithProject(id))
```

Can we return an error if `opts == nil` since theses options are mandatory.